x11-terms/cool-retro-term/cool-retro-term-1.2.0.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg
+
+QTW_PN="qmltermwidget"
+QTW_PV=63228027e1f97c24abb907550b22ee91836929c5
+QTW_P="${QTW_PN}-${QTW_PV}"
+
+DESCRIPTION="Terminal emulator with an old school look and feel"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term/"
+
+SRC_URI="
+	https://github.com/Swordfish90/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/Swordfish90/${QTW_PN}/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
+"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+RESTRICT="test"
+
+DEPEND="
+	dev-qt/qtdeclarative:5[localstorage]
+	dev-qt/qtgraphicaleffects:5
+	dev-qt/qtquickcontrols:5[widgets]
+	dev-qt/qtsql:5
+	dev-qt/qtwidgets:5
+"
+
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+
+	(
+		rmdir "${QTW_PN}" &&
+		mv "${WORKDIR}/${QTW_P}" "${QTW_PN}"
+	) || die "${QTW_PN} initializing failed."
```

Which part of the initialization failed? You shouldn't use `( )` here and just do these steps as they come, adding error-handling to each individually.
```suggestion
	rmdir "${QTW_PN}" || die
	mv "${WORKDIR}/${QTW_P}" "${QTW_PN}" || die
```